Readability.js

```diff
@@ -1082,27 +1083,47 @@ Readability.prototype = {
       if (this._debug)
         this.log("Article content after paging: " + articleContent.innerHTML);
 
+      var parseSuccessful = true;
+
       // Now that we've gone through the full algorithm, check to see if
       // we got any meaningful content. If we didn't, we may need to re-run
       // grabArticle with different flags set. This gives us a higher likelihood of
       // finding the content, and the sieve approach gives us a higher likelihood of
       // finding the -right- content.
       if (this._getInnerText(articleContent, true).length < this._wordThreshold) {
+        parseSuccessful = false;
```

Comment:
So what I would do is move the result of the `_getInnerText().length` text call to a variable before the `if` statement:

```js
var textLength = this._getInnerText(articleContent, true).length;
if (textLength < this._wordThreshold) {
```